README.md

```diff
@@ -623,9 +677,105 @@ let {
   }
 );
 
+initializeM = initialize;
 C.prototype.m = method;
 ```
 
+#### Class Init Accessor Decorators
+
+```ts
+type ClassGetterDecorator = (value: Function, context: {
+  kind: "init-getter";
+  name?: string | symbol;
+  access?: { get?(): unknown };
+  isStatic: boolean;
+  isPrivate: boolean;
+  defineMetadata(key: string | symbol | number, value: unknown);
+}) => {
+  get?: Function,
+  initialize?: (value: Function) => Function
+} | void;
+
+type ClassSetterDecorator = (value: Function, context: {
+  kind: "init-setter";
+  name?: string | symbol;
+  access?: { set?(value: unknown): void };
+  isStatic: boolean;
+  isPrivate: boolean;
+  defineMetadata(key: string | symbol | number, value: unknown);
+}) => {
+  set?: Function,
+  initialize?: (value: Function) => Function
+} | void;
+```
+
+Like accessor decorators, init-getter and init-setter decorators receive the original getter/setter defined on the prototype as the value being decorated. They can optionally return a new method and an initializer function. The new method, if present, is defined in place of the original method on the prototype, and the initializer function, if present, is called when during the construction of the class instance.
+
+Further extending the `@logged` decorator, we can make it handle init-methods as well, logging both whenever an instance of the class is initialized and whenever it is called.
+
+```js
+function logged(value, { kind, name }) {
+  if (kind === "init-accessor") {
```

Comment:
I’m a bit confused here - why are there both setter/getter, and also accessor? When would i get “getter” versus “accessor”?

Reply:
That was a typo, it should have been `init-getter`